utilities/AFQ_mrtrixInit.m

```diff
@@ -32,42 +32,65 @@
 % For details: 
 % http://www.brain.org.au/software/mrtrix/tractography/preprocess.html
 % 
-
-if notDefined('mrtrix_folder'), mrtrix_folder = 'mrtrix'; end
```

Comment:
It didn't work for me and I did a quick fix to make it work. If it is working for you please don't accept the change. Please notice that structure is different to the one found in vistasoft.